### DISCUSSION TITLE
🚀 Allow member has_role tag role_id parameter to take and/or

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#3578] [Feature Requests]
### Suggest an idea for ExpressionEngine

I’m trying to switch from member_group conditions like

``{if member_group == "1" || member_group == "6"}``

to

``{exp:member:has_role role_id="1|6"}``

But the role_id param treats it like a string.  At the least, I'd think we want to allow a pipe.  There's really no good way to replace that common member group conditional in the above.  Adding & and NOT would be nice too.  But the pipe is a really, really common use.

**Comment by intoeetive** [comment]
This will be implemented in 7.4 with https://github.com/ExpressionEngine/ExpressionEngine/pull/2817
